.github/workflows/on-push-to-main.yml

```diff
@@ -21,61 +25,6 @@ jobs:
       - name: Lint (rustfmt and clippy)
         run: make lint
 
-#  release:
-#    runs-on: ubuntu-latest
-#    outputs:
-#      version: ${{ steps.release.outputs.release }}
-#    steps:
-#      - uses: actions/checkout@v3
-#      - name: Set release
-#        id: semrel
-#        uses: go-semantic-release/action@v1
-#        with:
-#          github-token: ${{ secrets.GITHUB_TOKEN }}
-#          allow-initial-development-versions: true
-#          force-bump-patch-version: true
-#
-#      - name: Output release
-#        id: release
-#        run: echo "::set-output name=release::${{ steps.semrel.outputs.version }}"
-#
-#  publish:
-#    runs-on: macos-latest
-#    needs: release
-#    env:
-#      MOMENTO_API_KEY: ${{ secrets.ALPHA_TEST_AUTH_TOKEN }}
-#      CARGO_REGISTRY_TOKEN: ${{ secrets.CARGO_REGISTRY_TOKEN }}
-#    steps:
-#      - uses: actions/checkout@v3
-#      - name: Install protoc
-#        run: ./scripts/install_protoc_osx.sh
-#
-#      - uses: actions-rs/toolchain@v1
-#        with:
-#          toolchain: stable
-#          components: rustfmt
-#          override: true
-#
-#      - name: Build
-#        run: cargo build --verbose
-#
-#      - name: Integration Tests
-#        run: cargo test --tests
```

Comment:
can add this build and test step back into the workflow if we want

Reply:
or I can leave it commented out but bring these lines back into the file if we want to keep them for reference or anything

Reply:
yeah i think maybe let's keep this stuff in, commented out, until we have our first successful release. Then i'd probably go ahead and add the tests back in to the build steps, but you don't need to add them now; it'll be faster for you to iterate without them.